cairo-vm-cli/src/main.rs

```diff
@@ -101,8 +103,15 @@ impl FileWriter {
     }
 }
 
-fn main() -> Result<(), Error> {
-    let args = Args::parse();
+fn run(args: impl Iterator<Item = String>) -> Result<(), Error> {
+    let args = Args::try_parse_from(args);
+    let args = match args {
+        Ok(args) => args,
+        Err(error) => {
+            println!("{error}");
```

Comment:
nit: I think this should print to stderr

Reply:
```suggestion
            eprintln!("{error}");
```